engine/class_modules/sc_death_knight.cpp

```diff
@@ -946,9 +950,9 @@ struct death_knight_t : public player_t {
     // conduit_data_t spirit_drain; Finesse trait, 70
 
     // Blood
-    // conduit_data_t debilitating_malady; // 123
-    // conduit_data_t meat_shield; // Endurance trait, 121
-    // conduit_data_t withering_plague; // 80
+    conduit_data_t debilitating_malady; // 123
+    conduit_data_t meat_shield; // Endurance trait, 121
+    conduit_data_t withering_plague; // 80
```

Comment:
So Debilitating Malady doesn't affect DRW's blood plague damage, but Withing Plague increases damage done by your DRW's Heart Strike? that's peculiar.

Reply:
Tell me about it.  From my testing notes, Withering Plague DRW damage amp is based on it's own BP.

DRW's blood boil is the one that amps the damage.
Casting BB -> DRW -> HS does not amp DRW-HS damage.
Casting DRW -> BB -> HS does amp DRW-HS damage

For debilitating malady, your BB applies a debuff to the mob which ramps the damage, but DRW's BB does not apply the debuff.  I could not observe any change in damage with DRW, but to be fair, the amounts are so small it might be a rounding error.  This ability ramps damage by 1% per stack, which you generally get 2, maybe 3 at most stacks due to BB cooldown.  With my personal BP I can see the damage go from 93 to 95, and at 3 stacks a 96.  DRW's BB is only 31 damage though.  With 2 stacks it should be enough to push it to 32, but I have only observed 31 from it.  Yes, this conduit actually only increases BP damage by 1% per stack at rank 1.